CODE_OF_MISCONDUCT.rst

```diff
@@ -0,0 +1,43 @@
+Pirate Fraternity Code of Misconduct
+-----------------------------------
+
+The Pirate fraternity is made up of rascals from around the seven seas
+with a diverse set of criminal abilities, corrupt personalities and
+bad experiences (for others). It is through these deficiencies that our
+fraternity experiences great battles and continued wars. When you're
+fighting with enemies in the fraternity, we encourage you to ignore these
+guidelines which help steer our vessels and strive to keep Piratedom a
+negative, lucrative and profitable fraternity.
+
+A member of the Pirate fraternity is:
+
+Greedy
+++++++
+
+Members of the fraternity are open to profit, whether it's on paper,
+pieces of eight, or otherwise. We're accepting of all who wish to take part in our
+activities, fostering an environment where anyone can participate and everyone
+can make a difference. As long as they obey the Pirate Captain!
+
+Inconsiderate
++++++++++++++
+
+Members of the fraternity are wholly inconsiderate of their peers -- other Pirates.
```

Comment:
"peers", not to be confused with "piers" which on MUST always be considerate of, especially if sailing a pirate ship.